js/map/loadLeafletMap.js

```diff
@@ -393,157 +369,145 @@ let deleteOnClick = () => {
+    /**
+     * loadProducts - Loads all products from database and each category creates a new accordion
+     *
+     * @returns {void}
+     */
+    loadProducts: () => {
+        fetch(
+            `${configuration.apiURL}/obj/all/local/${id}?token=${token}`
+        )
+            .then((response) => {
+                return response.json();
+            })
+            .then((json) => {
+                if (!json.error) {
+                    objectData = json;
+                    let list = document.getElementsByClassName('obj-list')[0];
+
+                    for (let i = 0; i < json.length; i++) {
+                        if (json[i].Kategori != undefined) {
+                            if (document.getElementsByClassName(json[i].Kategori).length == 0 &&
+                                json[i].Kategori != "Pump") {
+                                list.innerHTML +=
+                                    `<button class="accordion desc">${json[i].Kategori}</button>
+							 <div class="panel"></div>`;
+
+                                let panels = document.getElementsByClassName('panel');
+                                let panel = panels[panels.length - 1];
+
+                                let object = document.createElement('div');
+
+                                object.innerHTML =
+                                    `<div class="obj-container">
+								<div id="${json[i].Modell}" class="obj ${json[i].Kategori}">
+									<img src="${json[i].Bild}"/>
+								</div>
+								<div class="obj-desc">${json[i].Modell}</div>
+							 </div>`;
+
+                                panel.appendChild(object);
+
+                                delete json[i].Bild;
+                                delete json[i].creatorID;
+                                delete json[i]._id;
+                                objectData.push(json[i]);
+                            } else if (json[i].Kategori != "Pump") {
+                                let elements = document.getElementsByClassName(json[i].Kategori);
+                                let panel = elements[0].parentElement.parentElement.parentElement;
+
+                                let object = document.createElement('div');
+
+                                object.innerHTML =
+                                    `<div class="obj-container">
+								<div id="${json[i].Modell}" class="obj ${json[i].Kategori}">
+							   		<img src="${json[i].Bild}"/>
+						   		</div>
+						   		<div class="obj-desc">${json[i].Modell}</div>
+							 </div>`;
+
+                                panel.appendChild(object);
+                            }
+                        }
+                    }
+
+                    accordions();
+                    show.activeObj();
+
+                    loadClickEvent();
+                    addPipeOnClick();
+                    addHouseOnClick();
+                } else {
+                    if (json.info == "token failed to validate") {
```

Comment:
Probably better to check the json.error parameter as it will always be true or false while the error message may vary.